### PR TITLE
added adme example

### PR DESCRIPTION
**Pull request by patrickhop**
Abstract:
"ADME in-silico methods have grown increasingly powerful over the past twenty years, driven by advances in machine learning and the abundance of high-quality training data generated by laboratory automation. Meanwhile, in the technology industry, deep-learning has taken off, driven by advances in topology design, computation, and data. The key premise of these methods is that the model is able to pass gradients back into the feature structure, engineering its own problem-specific representation for the data. Graph Convolutional Networks (GC-DNNs), a variation of neural fingerprints, enable true deep-learning in the chemistry domain. We use this new approach to build human plasma protein binding, lipophilicty, and humans clearance models that significantly outperform random-forests, support-vector-regression, and pyramidal neural networks."

I'd like to push the benchmark for our ADME paper (should show up on the arxiv in a day or two) into the DeepChem codebase. Predicting plasma-protein-binding and clearance are hard prediction problems that classical methods don't excel at. Good public ADME data is hard to find, but AstraZeneca has a few high-quality datasets on ChEMBL that are big enough to be usable.

Excited to hear any and all feedback you guys have for me.

### CODE REVIEW

**Comment by rbharath** [comment]
@patrickhop Thanks for the contributed dataset! Would you mind adding a README file that explains the lay-out of the data and its contents?

It looks like `yapf` is complaining. You should be able to install `yapf` locally with `pip install yapf` and then run `yapf` on changed files with `yapf -i file.py`

**Comment by rbharath** [comment]
@patrickhop LGTM. Will merge in once tests pass. Thanks for the contribution!

**Comment by miaecle** [comment]
@patrickhop Hi, thanks for contributing these datasets, look really nice! As I am working on the update of dc.molnet module, would you mind if I add these three datasets into the module collection?

**Comment by patrickhop** [comment]
@miaecle sounds good to me :)
